app/js/model/model.js

```diff
@@ -0,0 +1,5 @@
+define(['backbone'],
```

Comment:
You are dealing with todo and todos. The repetition in the path should also be a hint that the name is not good.